lib/driver.rb

```diff
@@ -39,9 +38,57 @@ def average_rating
     def add_trip(trip)
       if trip.class != Trip
         raise ArgumentError.new("Can only add trip instance to trip collection")
+      elsif trip.end_time == nil
+      change_status
       end
```

Comment:
Since `change_status` toggles the status, if you add two incomplete trips the status will go back to `:AVAILABLE`.